apps/admin/src/services/faq/mutations.ts

```diff
@@ -4,7 +4,23 @@ import { PostFaqReq } from '@/types/faq/remote';
 import { useMutation } from '@tanstack/react-query';
 import { useRouter } from 'next/navigation';
 import { toast } from 'react-toastify';
-import { postFaq } from './api';
+import { deleteFaq, postFaq } from './api';
+
+export const useFaqDeleteMutation = (id: number) => {
```

Comment:
```suggestion
export const useDeleteFaq = (id: number) => {
```

api 엔드포인트가 deleteFaq니깐 useDeleteFaq로 가는건 어떨까요?..

Reply:
그게 좋겠네요